.github/workflows/release.yaml

```diff
@@ -1,53 +0,0 @@
-name: Publish Python Package
```

Comment:
I've completely deleted this file on the basis I don't think this has ever run to completion. We've never pushed a version of this package to PyPI https://pypi.org/project/dc_design_system/ Also even if it had worked at one point, the auth would now be broken.
Lets just bin this for now and come back to it when we look at fixing up package publishing and trusted publisher workflow across packages.